server/routes/probationPractitionerReferrals/interventionProgressPresenter.ts

```diff
@@ -43,7 +43,7 @@ export default class InterventionProgressPresenter {
         sessionNumber: appointment.sessionNumber,
         appointmentTime: DateUtils.formatDateTimeOrEmptyString(appointment.appointmentTime),
         tagArgs: this.tagArgs(appointment),
-        linkHtml: appointment.sessionFeedback?.attendance ? `<a class="govuk-link" href="#">View</a>` : '',
+        linkHtml: appointment.sessionFeedback?.submitted ? `<a class="govuk-link" href="#">View</a>` : '',
```

Comment:
hooray for unit tests 🎉

Reply:
This is a functional change - would it be too much of a pain to split it out into its own commit?

Reply:
since `attendance` is no longer optional in the updated model, this change is nessecary here. we could alter this to look something like `sessionFeedback?.attendance.attended != null`, but then i don't think it's really correct.

happy to split the commit out, but fixing getting the tests green in both commits will be a pain. can i just expand on the commit message?